spec/features/admin/disable_enable_user_feature_spec.rb

```diff
@@ -12,8 +12,29 @@
   end
 
   context "authorised" do
+    # Disabling a user is tested below but it veers off into an
+    # authentication test. Here we are concerned with toggling the
+    # user's status
+    scenario "Disable / Enable a user" do
+      system_user = create(:user).tap { |u| u.add_role :system }
+      other_user = create(:user).tap { |u| u.add_role :admin_agent }
+
+      login_as system_user
+      visit admin_users_path
+
+      click_link "Disable"
+      fill_in "Comment", with: "User has left the company"
+      click_button "Disable user"
+      click_link "Show all users"
+      expect(page).to have_css(".user-status", text: "Disabled")
+
+      click_link "Enable"
+      expect(page).to have_css("h1", text: "You are about to enable the user #{other_user.email}")
+      click_button "Yes, continue"
+      expect(page).to have_css(".user-status", text: "Enabled")
```

Comment:
Is there a way to check that it's the other user who is displaying "enabled", rather than just the system user?

Reply:
Yes - good idea!